<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RCConditionItemEditorDialog</class>
 <widget class="QDialog" name="RCConditionItemEditorDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="1">
    <widget class="CRCFieldsComboBox" name="cmbField"/>
   </item>
   <item row="1" column="1">
    <widget class="CRBComboBox" name="cmbConditionType"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblField">
     <property name="text">
      <string>Поле</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblConditionType">
     <property name="text">
      <string>Тип условия</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblValue">
     <property name="text">
      <string>Значение</string>
     </property>
     <property name="buddy">
      <cstring>edtValue</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="CRBComboBox" name="cmbValueType"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblValueType">
     <property name="text">
      <string>Тип Значения</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>73</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="valueLayout">
     <item>
      <widget class="QLineEdit" name="edtValue">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CRCFieldsComboBox" name="cmbValueField">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CRCComboBox" name="cmbParams">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="edtDate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="edtValueInt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="edtValueDouble">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CRCFieldLineEdit" name="edtCustom"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRCFieldsComboBox</class>
   <extends>QComboBox</extends>
   <header>Reports/ReportsConstructor/RCComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CRCComboBox</class>
   <extends>QComboBox</extends>
   <header>Reports/ReportsConstructor/RCComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CRCFieldLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Reports/ReportsConstructor/RCLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RCConditionItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RCConditionItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
